sdk/web-pubsub/web-pubsub/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "../../../tsconfig.package",
   "compilerOptions": {
     "outDir": "./dist-esm",
+    "lib": ["DOM"],
```

Comment:
IIRC this forces downstream consumers of your package to also use this lib setting, which sometimes upsets Node devs. 

Perhaps we can update `"@types/node": "^8.0.0",` to 10.x?